frappe/public/scss/desk/version.scss

```diff
@@ -0,0 +1,33 @@
+.version-info {
+	overflow: auto;
+
+	pre {
+		border: 0px;
+		margin: 0px;
+		background-color: inherit;
+	}
+
+	.table {
+		background-color: inherit;
+	}
+
+	.success {
+		background-color: #dff0d8 !important;
+	}
+
+	.danger {
+		background-color: #f2dede !important;
+	}
```

Comment:
Better to use available CSS variables for colors

Reply:
Done